services/community-site/src/pages/Staking/Modal/Undelegate.tsx

```diff
@@ -53,6 +53,16 @@ const Undelegate = ({ validator, onSuccess, onFinish }: UndelegateProps) => {
   ) => {
     event.preventDefault();
 
+    if (
+      parseFloat(undelegationTotal) < parseFloat(totalDelegation) &&
```

Comment:
Tho I'm wondering why float instead of Number here

Reply:
Because the with `Number()`  function, any trailing decimals will be lost